backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -65,6 +66,10 @@ Users.validateUpdateData = async (req, res, next) => {
 				}
 				return true;
 			}),
+		company: Yup.string(),
```

Comment:
We should impose lengths. We should never allow user to give us indefinite number of characters

use `strings.title`